src/app/components/shell/router-helpers/page-routes.tsx

```diff
@@ -79,25 +78,27 @@ export function OtherPageRoutes() {
     }
 
     // Some pages have no page data in the CMS!
-    if (
-        [
-            'adopters',
-            'adoption',
-            'blog',
-            'campaign',
-            'confirmation',
-            'institutional-partnership-application',
-            'interest',
-            'renewal-form',
-            'separatemap'
-        ].includes(dir)
-    ) {
+    if (isNoDataPage(dir)) {
         return <ImportedPage name={dir} />;
     }
 
     return <RouteAsPortalOrNot />;
 }
 
+export function isNoDataPage(dir: string) {
```

Comment:
Handling these in both portal and non-portal routes was the main issue I had to fix.